core/src/main/java/bio/terra/pearl/core/model/survey/AnswerFormat.java

```diff
@@ -0,0 +1,6 @@
+package bio.terra.pearl.core.model.survey;
+
+public enum AnswerFormat {
+    NONE,
```

Comment:
I debated `TEXT` as the default but that might not always be true (e.g., if it's in the number column). So I felt `NONE` was more appropriate for a standard "unformatted" answer.